oat-sa/extension-tao-itemqti
# Handle async PCI destroy callback

Required modifications to handle async destroy callback in PCI. Pull request: https://github.com/oat-sa/extension-tao-itemqti-pci/pull/135

To test:
- checkout the branch from the mentioned pull request for `qtiItemPci`
- checkout the branch from this pull request
- create a new delivery with an audio PCI
- trigger a destroy step (for example navigating to the next item)

Expected result: the audio PCI should get destroyed without any error.
